pom.xml

```diff
@@ -138,7 +138,7 @@
             </goals>
             <configuration>
               <rules>
-                <bansRuntimeDependency implementation="io.quarkus.enforcer.BansRuntimeDependency" />
+                <bansRuntimeDependency implementation="io.quarkus.enforcer.BansRuntimeDependency"/>
```

Comment:
Because this PR was created by a script, it would be nice to merge this to remove any false positives on future PRs